Harvest.Core/Models/History/RateHistoryModel.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Harvest.Core.Domain;
+
+namespace Harvest.Core.Models.History
+{
+    public class RateHistoryModel
+    {
+        public static RateHistoryModel CreateFrom(Rate rate)
```

Comment:
why sometimes a static and sometimes you use the ctor?  Like `RateHistoryModel.CreateFrom(xyz)` vs sometimes you do `new RateHistoryModel(xyz)`?

Reply:
For some, like User, they will only ever be instantiated from a property of a parent domain entity. Since those could be null based on whether they were included, I thought it would look cleaner than a bunch of ternary operators.

Reply:
makes sense